gc/base/ParallelSweepChunk.hpp

```diff
@@ -89,7 +89,7 @@ class MM_ParallelSweepChunk : public MM_BaseNonVirtual
 	 */	
 	MMINLINE void clear()
 	{
-		memset(this, 0, sizeof (*this));
+		*this = MM_ParallelSweepChunk();
```

Comment:
Are all of the fields in MM_ParallelSweepChunk properly initialized in the constructor?

Reply:
The constructor is not initializing these fields:
```c++
#if defined(OMR_GC_CONCURRENT_SWEEP)
	MM_ParallelSweepChunk *_nextChunk;
	uintptr_t _concurrentSweepState;
#endif /* OMR_GC_CONCURRENT_SWEEP */
```

Reply:
Woops, looks like two were missing:
```c++
#if defined(OMR_GC_CONCURRENT_SWEEP)
	MM_ParallelSweepChunk *_nextChunk;
	uintptr_t _concurrentSweepState;
#endif /* OMR_GC_CONCURRENT_SWEEP */
```